client/components/edit/datastream/DatastreamControlButton.tsx

```diff
@@ -16,7 +15,17 @@ const Icons = {
     Delete: <Delete />,
 };
 
-const DatastreamControlButton = ({ modalState, datastream, disabled }) => {
+interface DatastreamControlButtonProps {
+    modalState: string;
+    disabled: boolean;
```

Comment:
I realize that order doesn't really matter, but I wonder if it would be more clear/readable if the order of the interface and the order of the extracted parameters matched up. For some reason, disabled and datastream are transposed here.

Reply:
Made change